packages/core/strapi/lib/utils/cron.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const shiftHours = (date, step) => {
+  const frequency = 24 / step;
+  const list = Array.from({ length: frequency }, (_, index) => index * step);
+  const hour = date.getHours();
+  return list.map((value) => (value + hour) % 24).sort((a, b) => a - b);
```

Comment:
i am assuming 24 expresses that there is 24 hours a day, can we have it stored in a variable that expresses it's meaning ? it would make it more readable and easier to understand

Reply:
I added the utility function I wanted to do in the first place. Everything should be clearer now. I'll also add some JSDoc, as well as some tests of the utility itself.

It's a cron expression shifter that could be reused everywhere we need to simulate an interval.